src/k8s-engine/client.js

```diff
@@ -360,6 +360,21 @@ class KubeClient extends events.EventEmitter {
     return this.#servers.get(namespace, endpoint, port)?.address()?.port;
   }
 
+  /**
+   * Return a list of all the pods with the given app-label in the specified namespace
+   * @param namespace
+   * @param endpointName
+   * @returns {Promise<*[{{ name: string, status: string }}]>}
+   */
+  async listPods(namespace, endpointName) {
```

Comment:
The term `endpointName` confused me on first pass. I had to read the code to understand what it was doing. Might there be a better name to use here?

Reply:
Existing code… maybe `podName`?

Reply:
Ahh, `endpointName` was used because there's another call to the `ListNamespacedEndpoints` method in the coreV1 API to get the list of pods. This method is an alternative way to get that information, so this name doesn't make sense.